Mainframe3270/x3270.py

```diff
@@ -13,7 +14,7 @@
 from .py3270 import Emulator
 
 
-class x3270(object):
+class X3270(object):
```

Comment:
I needed to change this name in order to mock the os_name in the unit tests. Since the only interface to the user is the `Mainframe3270` class, I do not consider this to be a breaking change.